modules/rtm.c

```diff
@@ -57,7 +57,7 @@ void RTM_Manage( void * Parameters )
     rtm_disable_payload_power();
 
     for ( ;; ) {
-        vTaskDelay(500);
+        vTaskDelay(pdMS_TO_TICKS(50));
```

Comment:
The current FreeRTOS tick time is set to 1 ms, so you should use `pdMS_TO_TICKS(500)` here.